docs/sources/mimir/set-up/helm-chart.md

```diff
@@ -0,0 +1,12 @@
+---
+description: Learn how to deploy Mimir with Helm.
+title: Deploy Mimir with Helm
+menuTitle: Deploy with Helm
+weight: 350
+---
+
+# Deploy Mimir with Helm
+
+The [`mimir-distributed` Helm chart](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/README.md) for Grafana Mimir and Grafana Enterprise Metrics allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
```

Comment:
I think pointing to the whole chart is more accurate in this case, plus github shows the contents for the README.md anyway:

```suggestion
The [`mimir-distributed` Helm chart](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/) for Grafana Mimir and Grafana Enterprise Metrics allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
```